oaknational/Oak-Web-Application
# LESQ-756: Focus after successful download

## Description

Music year: 1998

- Adds tab index to download button

# Acceptance Criteria:

- [ ]  When using a keyboard, after a download, focus should be moved to the confirmation page (probably the “back to lesson” link?

## Issue(s)

Fixes #LESQ-756

## How to test

1. Go to https://deploy-preview-2452--oak-web-application.netlify.thenational.academy
2. Navigate to any downloads page and after downloading assets, press tab
3. Back to lesson button should be the first element tabbed onto

## Screenshots

How it should now look:

https://github.com/oaknational/Oak-Web-Application/assets/91190841/eb144771-84f2-43f5-a5b3-8ae50ed3cae7


## Checklist
